tcf_website/templates/search/search.html

```diff
@@ -14,6 +14,14 @@ <h1>Search: {{query}}</h1>
 
     <hr>
 
+    {% if courses_error %}
+    <div class="container">
+        <h4>Searching courses is down right now!</h4>
+        <li>{{courses_error}}</li>
```

Comment:
Should we surface the error code to users?

Reply:
probably not, and not passing the error message will simplify the views code as well